src/app/newcomers/newcomers.component.scss

```diff
@@ -37,6 +37,12 @@
   }
 }
 
+@media only screen and (max-width: 1024px) {
```

Comment:
I noticed the image doesn't show on mobile - is that the design?

Reply:
Yeah. That was suggested by @Janiceilene in All teams meeting. What are your views about it?